pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -143,10 +143,12 @@ self: super:
     outputs = [ "out" "dev" "devdoc" ];
   });
 
-  # See https://bugs.freedesktop.org/show_bug.cgi?id=47792
-  # Once the bug is fixed upstream, this can be removed.
   luit = super.luit.overrideAttrs (attrs: {
+    # See https://bugs.freedesktop.org/show_bug.cgi?id=47792
+    # Once the bug is fixed upstream, this can be removed.
     configureFlags = [ "--disable-selective-werror" ];
+
+    buildInputs = attrs.buildInputs ++ [libiconv];
```

Comment:
Is libiconv null on other platforms?

Reply:
No.  Anyway, it's small &ndash; see `glibcIconv` definition in `all-packages.nix`

Reply:
Yeah, it doesn't have to be conditional anymore. (see https://github.com/NixOS/nixpkgs/pull/37012)